packages/venia-ui/lib/components/CategorySort/categorySort.css

```diff
@@ -0,0 +1,68 @@
+.root {
+    position: relative;
+}
+
+.sortButton {
+    margin-left: 0.6rem;
+    display: inline-block;
+    padding: 0.8rem 2rem;
+    border: 1px solid black;
+    border-radius: 3px;
+    color: black;
+    outline: none;
+}
+
+.menu {
+    position: absolute;
+    z-index: 2;
+    top: 110%;
+    right: 0;
+    min-width: 10rem;
+    margin: 0.125rem 0 0;
+    font-size: 1rem;
+    color: black;
+    text-align: left;
+    list-style: none;
+    background-color: #fff;
+    background-clip: padding-box;
+    border: 1px solid rgb(var(--venia-grey-dark));
+    border-radius: 0.25rem;
+    box-shadow: 0 1px 3px #3f3b4a26;
```

Comment:
Clever use of extended hex, but let's stick with `rgba()`. Also, we do try to avoid arbitrary colors in our CSS. 😅 

Have a look at [`autocomplete.css`](https://github.com/magento/pwa-studio/blob/develop/packages/venia-ui/lib/components/SearchBar/autocomplete.css) for an example of the shadow for a dropdown menu.